### PR TITLE
Fix tools & add make tidy

### PR DESCRIPTION
**Pull request by natalieparellano**
Tools needs to be pinned to older x/sys as well.

Signed-off-by: Natalie Arellano <narellano@vmware.com>

### CODE REVIEW

**Comment by natalieparellano** [comment]
Should we automatically run `make tidy` when we test / format / lint?

File: Makefile [hunk 1]
```diff
@@ -203,6 +203,10 @@ format: install-goimports
 	@echo "> Formating code..."
 	$(if $(shell goimports -l -w -local github.com/buildpacks/lifecycle .), @echo Fixed formatting errors. Re-run && exit 1)
 
+tidy:
```
Comment (yaelharel) [inline Makefile:206]: I think it's worth adding this to `make unit/all/acceptance`.
Or maybe we can create another target that is running `lint format tidy` and use it when running `make unit/all/acceptance`.

Comment (natalieparellano) [reply Makefile:206]: Added to test targets

**Review by yaelharel — APPROVED** [review]

**Review by jabrown85 — APPROVED** [review]
This LGTM - I could see omitting `tidy` from `unit` and `acceptance` for local dev experience speed reasons, but this aligns with `format` and `lint` so 👍

**Pull request merged**